app/controllers/collections_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CollectionsController < ApplicationController
+  def index
+    if (@user = User.find_by(username: params[:username]))
+      @collections = Collection.where(user_id: @user.id)
+    elsif (@user = Organization.find_by(slug: params[:username]))
+      @collections = Collection.where(organization_id: @user.id)
+    end
```

Comment:
Although we do have the concept of "series optionally belonging to an organization" there's currently no code path in which an organization can create a series, so I would probably drop this altogether and just load: `user.collections` for now

Reply:
I would also order them by descending creation order, or alphabetical title, you choose :D

Reply:
Ok, will do!